app/templates/account/profile.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui grid container">
+        <div class="ten wide computer sixteen wide mobile centered column">
+            <h2 class="ui dividing header">FirstName LastName</h2>
+            
+                <div class="ui one column grid">
+                    
+                    <div class="three column row">
+                        <div class="column"><h3>Military Affiliation</h3></div>
+                        <div class="column"><h3>Birthday/Age</h3></div>
+                        <div class="column"><h3>Location</h3></div>
+                    </div>
+                    
+                    <div class="left aligned column">
+                        <h3>Bio</h3>
+                        <h3>Issues I care about</h3>
+                        <h3>Links</h3>
+                   
+                    </div>    
+                    
```

Comment:
There are a ton of extra blank lines in this html section. Let's take all those away.